docs/configuration-reference/platforms/baremetal.md

```diff
@@ -54,6 +54,17 @@ cluster "bare-metal" {
 
```

Comment:
I think docs changes should be part of `baremetal: Ability to add CLC snippets in lokocfg` commit. What do you think?

Reply:
I like to separate out documentation commits from the code changes.

Reply:
I also think docs should be committed together with feature code, so we keep things atomic.